### PR TITLE
Todo hunting

### PR DESCRIPTION
**Pull request by DantElemer**
Some TODO remained but they are either solved on other branch or should be solved only right before finishing the project.
I tested it, so probably no errors were introduced, but many parts were modified (even if only a little bit), so it is not sure.

### CODE REVIEW

File: src/Client/Player/Player.py [hunk 1]
```diff
@@ -9,14 +9,15 @@
...
 class PicFile(Enum):
     ORANGE = "img/orange_test_image.png"
     APPLE = "img/apple_test_image.png"
 
 
 class Player(pg.sprite.Sprite):
-    SCREEN_HEIGHT = 600  # TODO use general constant
     GREEN = (0, 255, 0)
     RED = (255, 0, 0)
     WHITE = (255, 255, 255)
```
Comment (marzsolt) [inline src/Client/Player/Player.py:23]: It was not explicitly mentioned in each row, but I would consider useful to use general constants for the colours as well. Won't you?

File: src/Client/Screen/Screen.py [hunk 2]
```diff
@@ -16,13 +16,13 @@
...
 
 class Screen:
     """ Screen - responsible for the main tasks on client side, such as drawing to display, etc. """
     # Definition of some constants
-    SCREEN_WIDTH = 800
-    SCREEN_HEIGHT = 600
-
     BLACK = (0, 0, 0)
     WHITE = (255, 255, 255)
     GREEN = (0, 255, 0)
```
Comment (marzsolt) [inline src/Client/Screen/Screen.py:28]: Again, colours... [?]

File: src/Server/PlayerAI/AppleAI.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 class AppleAI(PlayerAILogic, AppleLogic):
     def __init__(self, player_id, terrain, game):
         super(AppleAI, self).__init__(player_id, terrain, game)
-        self._attack_range = 150  # TODO compute it normally
+        self._attack_range = 150
 
     def _attack(self):
         force_of_jump = self._calculate_attack_force(self.pos.x, self.pos.y + 100) # TODO egyelőre csak felfele ugrik egyet
```
Comment (marzsolt) [inline src/Server/PlayerAI/AppleAI.py:12]: I can see an unresolved TODO over there :D

File: src/Client/Player/Player.py (hunk 1 above)
Comment (DantElemer) [reply src/Client/Player/Player.py:23]: It occured to me as well. So be it.
Oh I just noticed that there is an FPS too. It's the most important for that to be general...

File: src/Server/PlayerAI/AppleAI.py (hunk 3 above)
Comment (DantElemer) [reply src/Server/PlayerAI/AppleAI.py:12]: "Some TODO remained but they are either solved on other branch or should be solved only right before finishing the project."

**Pull request merged**